content/en/getting-started/installation.md

```diff
@@ -58,10 +61,27 @@ If you have problems with permissions in MacOS X Sierra, install with `python3 -
 
 {{< tab header="Linux" >}}
 <p>
-You can just download the respective binary for your architecture directly:<br>
+You can download the respective binary for your architecture directly:<br>
 {{< cli-binary-download os="linux" >}}
 </p>
 <p>
+or use this <code>curl</code> command:
+</p>
+<p>
+For <i>amd64</i>:
+{{< command >}}
+$ curl -Lo ~/Downloads/localstack-cli-{{< localstack-latest-version >}}-linux-amd64-onefile.tar.gz \
+    https://github.com/localstack/localstack-cli/releases/download/v{{< localstack-latest-version >}}/localstack-cli-{{< localstack-latest-version >}}-linux-amd64-onefile.tar.gz
+{{< / command >}}
+</p>
+<p>
+Or <i>arm64</i>:
```

Comment:
nit: I think we should use the same architecture references as we use for the download buttons.
```suggestion
Or `ARM64`:
```